### ISSUE TITLE
How to get the view_mvn and light_mvn files of my datasets?

### ISSUE THREAD

**Issue by RainCoderJoe** [XingangPan/GAN2Shape#17]
Hi.
Thank you for your works!
These view_light files only include car, celeba, face and cat. How can I get the view_mvn and light_mvn of my datasets?

**Comment by XingangPan** [comment]
@RainCoderJoe Hi, you can use the method in this repo https://github.com/elliottwu/unsup3d to estimate the mean and covariance of viewpoint and lighting variations.

**Comment by RainCoderJoe** [comment]
In other words, I need to train unsup3d on my datasets, and then estimate viewpoint and lighting of all images, and finally get the mean and covariance of viewpoint and lighting?

**Comment by XingangPan** [comment]
Yes, that's what I mean. Besides, you may also try using the provided view_mvn files and see if they work. Usually, the view_mvn does not have to be very accurate.

**Comment by RainCoderJoe** [comment]
Thank you for your help!
